Correct limit_by service for rate-limiting plugin (#4626)
Kong/docs.konghq.com: app/_hub/kong-inc/rate-limiting/_index.md

      default: '`consumer`'
      datatype: string
      description: |

>         The entity that is used when aggregating the limits: `consumer`, `credential`, `ip`, `service`, `header`, and `path`.
>         If the entity value for aggregating the limits cannot be determined, the system falls back to `ip`.
>         
>         If the `service` value is chosen, the `service.id` or `service.name` configuration must be provided if you're adding the plugin to a service through the top-level `/plugins` endpoint.
>         
>         If the `header` value is chosen, the `header_name` configuration must be provided.
>         
>         If the `path` value is chosen, the `path` configuration must be provided.

Reviewer: ```suggestion
        The entity that is used when aggregating the limits. Available values are:
        - `consumer`
        - `credential`
        - `ip`
        - `service` (The `service.id` or `service.name` configuration must be provided if you're adding the plugin to a service through the top-level `/plugins` endpoint.)
        - `header` (The `header_name` configuration must be provided.)
        - `path` (The `path` configuration must be provided.)
        If the entity value for aggregating the limits cannot be determined, the system falls back to `ip`.
```
